compose/project.go

```diff
@@ -177,7 +177,7 @@ func newCoreServiceProject(cfg *config.CloudConfig, network bool) (*project.Proj
 
 	go func() {
 		for event := range projectEvents {
-			if event.EventType == project.EventContainerStarted && event.ServiceName == "network" {
+			if event.EventType == project.EventContainerStarted && event.ServiceName == "ntp" {
```

Comment:
Because HTTPS certs appear "not yet valid"

On Fri, Apr 1, 2016, 6:41 PM Darren Shepherd <notifications@github.com>
wrote:

> In compose/project.go
> <https://github.com/rancher/os/pull/825#discussion_r58286268>:
>
> > @@ -177,7 +177,7 @@ func newCoreServiceProject(cfg *config.CloudConfig, network bool) (*project.Proj
> >
> >  	go func() {
> >  		for event := range projectEvents {
> > -			if event.EventType == project.EventContainerStarted && event.ServiceName == "network" {
> > +			if event.EventType == project.EventContainerStarted && event.ServiceName == "ntp" {
>
> @imikushin <https://github.com/imikushin> why ntp? why not
> "wait-for-network"?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/rancher/os/pull/825/files/0f6cf07786d6b5675be252d55164bf59be1102a9#r58286268>
>